src/profile-logic/marker-data.js

```diff
@@ -993,6 +1021,10 @@ export function filterRawMarkerTableToRangeWithMarkersToDelete(
   oldMarkerIndexToNew: Map<IndexIntoRawMarkerTable, IndexIntoRawMarkerTable>,
 } {
   const newMarkerTable = getEmptyRawMarkerTable();
+  const newThreadId = [];
+  if (oldMarkerTable.threadId) {
+    newMarkerTable.threadId = newThreadId;
```

Comment:
I believe this code is not really used: this function is used by the sanitization, and we never pass merged threads to this function.